# goal
# possible modes:
## 0 -> Enumeration
## 1 -> Classification
## 2 -> Comparison
uint8 mode
string[] expected_tags
string table_id
---
# result
int8 n_found_tags
string[] found_tags
bool match
---
# feedback
string step